src/courses/mappers/02.md

```diff
@@ -0,0 +1,128 @@
+:::
+
+:::details NIST RMF
+The NIST Risk Management Framework (NIST RMF) provides "a process that integrates security, privacy, and cyber supply chain risk management activities into the system development life cycle."
+
+More information can be found [here](https://csrc.nist.gov/projects/risk-management/about-rmf).
+:::
+
+:::::details NIST SP 800-53
+- The NIST Special Publication 800-53 (**NIST SP 800-53**) "provides a catalog of security and privacy controls for information systems and organizations to protect organizational operations and assets, individuals, other organizations, and the Nation from a diverse set of threats and risks."
+
+More information can be found [here](https://csrc.nist.gov/pubs/sp/800/53/r5/upd1/final).
+
+:::details NIST SP 800-53 Excerpt
+AC-17 REMOTE ACCESS
+Control:
+a. Establish and document usage restrictions, configuration/connection requirements, and
+implementation guidance for each type of remote access allowed; and
+b. Authorize each type of remote access to the system prior to allowing such connections.
+:::
+:::::
+
+:::details DISA
+The Defense Information Systems Agency (DISA) is a U.S. agency conducting "DODIN (Department of Defense information networks) operations to enable lethality across all warfighting domains in defense of our nation."
+
+More information can be found [here](https://www.disa.mil/).
+:::
+
+:::::details CCIs
+Control Correlation Identifiers (**CCIs**) are "standard identifiers and descriptions by DISA that aim to correlate high-level policy expressions and low-level technical implementations of security requirements." They are analogous to NIST SP 800-53s in that they both provide security and privacy controls.
+
+More information can be found [here](https://public.cyber.mil/stigs/cci/).
+
+:::details CCI Excerpt
+CCI-000002: Disseminate the organization-level; mission/business process-level; and/or system-level access control policy that addresses purpose, scope, roles, responsibilities, management commitment, coordination among organizational entities, and compliance to organization-defined personnel or roles.
+
+References:
+- NIST: NIST SP 800-53 (v3): AC-1 a
+- NIST: NIST SP 800-53 Revision 4 (v4): AC-1 a 1
+- NIST: NIST SP 800-53 Revision 5 (v5): AC-1 a 1 (a)
+- NIST: NIST SP 800-53A (v1): AC-1.1 (iii)
+:::
+:::::
+:::::::
+
+## What Else?
+
+You can read more about OHDF [here](https://saf.mitre.org/framework/normalize).
+
+## Knowledge Check
+
+:::details What is OHDF?
+OHDF is a security data format used to normalize generated data exports from various security tools into a single common data format usable by the SAF tool suite.
+:::
+
+:::details What are some uses of OHDF?
+Some uses include:
+
+- Aggregating security data formats into a single standard format.
+
+- Facilitating the transformation/transport of data across different processes and technologies.
+
+- Enhancing security data with relevant compliance standards.
```

Comment:
change: - Enhancing security data with relevant **compliance** standards.
to: - Enhancing security data with relevant **security** standards.

Reply:
Addressed.